src/main/java/com/google/sps/servlets/GetApiKeyServlet.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.GetConfigProperties;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that return the API Key. */
+@WebServlet("/apikey")
```

Comment:
I don't think we want a servlet that's passing the key to the client, we want to try and use the api key on the server side so that it's hidden from the client so they can't steal our key and spend all our money.
If you do need it on the client side we look into ways to obfuscate it or not store it locally but I think you should be able to just fetch the things you need on the server side and pass it up to the client.